.github/workflows/changelog_check.yml

```diff
@@ -0,0 +1,16 @@
+Name: Changelog-Checks
+on: 
+  pull_request:
+    branches-ignore:
+      -master
+      -release-*
```

Comment:
I think we can likely exclude the `branches-ignore` key, as we probably won't put up a PR for a branch with the name `master` or `release-*` against any other branch. Leaving just the empty `pull_request:` key should just run the changelog test against any and all pull requests, which is already an improvement over [the buildkite config](https://github.com/matrix-org/pipelines/blob/e1ae0fb309e88345fb4903d5c4c72b19a47e62a6/sygnal/pipeline.yml#L10).

Reply:
Thanks for the notes-everything should be fixed now! I also was not aware of the request review button, thanks so much for telling me about that.